### DISCUSSION TITLE
Error creating app from budibase template

### DISCUSSION THREAD

**Discussion by applicationuser** [Budibase/budibase#8427] [Help]
Error creating app from budibase template

Budibase cloud

![image](https://user-images.githubusercontent.com/42922833/198346887-62fb1a24-c876-40e4-98bc-23614eca7521.png)

Error details from browser console browser 

Response

![image](https://user-images.githubusercontent.com/42922833/198345807-786dfa46-982b-4c59-9f7b-dbb1e3962535.png)

Payload 
![image](https://user-images.githubusercontent.com/42922833/198345841-61f7f9a1-2e68-437f-841d-0375f8076813.png)

**Comment by jonnymccullagh** [comment]
I tried to recreate your issue. I registered with budibase cloud using a google account, then created an app from a template. I chose the Agency Client Template with a URL of /agency-client-template and the app created fine. I repeated created another app from Agency Client Template at the url of /agency-client-template-1 and that created another app.
Could you try again?

**Reply by applicationuser** [reply]
Thank You @jonnymccullagh 

I was able to import app by going through steps of create new app and selecting template

Below are the steps that would give error for importing template.

->after login , navigate to Apps page. 
-> Click on Templates
-> Go to section of the app you want to import
-> Click on details , Template details opens in separate tab
-> Click on 'Use this template' -  Error received that app cannot be imported.

![image](https://user-images.githubusercontent.com/42922833/198674040-f3092460-2ccf-41c5-81c3-126dfcd0b895.png)

![image](https://user-images.githubusercontent.com/42922833/198674245-75ab0cbf-3335-4438-aad7-24eed448f0cd.png)

![image](https://user-images.githubusercontent.com/42922833/198674396-c1f3dea6-2656-483c-8939-575de26e7638.png)

**Reply by davidir** [reply]
I created an account to try in mind to use in the future with our clients and we have the same problem.

When you are in this page you're unable to dismiss o delete de installation.

David
Keliam.com

**Reply by jonnymccullagh** [reply]
@applicationuser thanks for that clarification. I was able to replicate your issue and have opened bug issue [8533](https://github.com/Budibase/budibase/issues/8533)
Thanks again for your feedback.
In the interim you should still be able to create apps from templates by staying within the budibase.app interface and clicking 'Use template'

**Reply by jonnymccullagh** [reply]
As per issue [8533](https://github.com/Budibase/budibase/issues/8533) this has now been fixed.
